### ISSUE TITLE
[BUG] Unable to start clustering monitoring in CRC-1.19

### ISSUE THREAD

**Issue by noelo** [code-ready/crc#1712]
### General information

  * OS: Linux
  * Hypervisor: KVM / Hyper-V / hyperkit
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: Baremetal-Server 

## CRC version
```bash
CodeReady Containers version: 1.19.0+94b0362
OpenShift version: 4.6.3 (embedded in executable)
```
  
## CRC status
```bash
[crc@CentOS-82-64-minimal ~]$ crc status --log-level debug
DEBU CodeReady Containers version: 1.19.0+94b0362
DEBU OpenShift version: 4.6.3 (embedded in executable)
Checking file: /home/crc/.crc/machines/crc/.crc-exist
DEBU About to run SSH command:
df -B1 --output=size,used,target /sysroot | tail -1
DEBU SSH command results: err: <nil>, output: 32720793600 13860200448 /sysroot
DEBU About to run SSH command with hidden output
DEBU SSH command succeeded
CRC VM:          Running
OpenShift:       Running (v4.6.3)
Disk Usage:      13.86GB of 32.72GB (Inside the CRC VM)
Cache Usage:     55.92GB
Cache Directory: /home/crc/.crc/cache
```

## CRC config
```bash
[crc@CentOS-82-64-minimal ~]$ crc config view
- memory                                : 61440
```

## Host Operating System
```bash
[crc@CentOS-82-64-minimal ~]$ cat /etc/os-release
NAME="CentOS Linux"
VERSION="8 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Linux 8 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:8"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-8"
CENTOS_MANTISBT_PROJECT_VERSION="8"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="8"
```

### Steps to reproduce

  1. Follow the directions in https://code-ready.github.io/crc/#starting-monitoring-alerting-telemetry_gsg to turn on monitoring
  2.  Gist with details https://gist.github.com/noelo/a28f71d0503f417bca75574918fac7c7
  3. This is related to https://github.com/code-ready/crc/issues/1648 where the issue I'm trying to solve is the inability to register a prometheus rule from another operator
  4. 

### Expected

Monitoring pods would start and other operators could register prometheus rules

### Actual

Nothing is spun up and no rules are registered

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
https://github.com/code-ready/crc/issues/1704 is tracking for this one and have a manual instructions to enable it, should be fixed in next release (1.20.0).

**Comment by anjannath** [comment]
Closing, this is fixed in the latest release v1.20.0 (4.6.6 bundle)
